vault/assets/service_checks.json

```diff
@@ -69,6 +69,6 @@
             "endpoint"
         ],
         "name": "OpenMetrics endpoint health",
-        "description": "Returns `CRITICAL` if the Agent is unable to connect to the OpenMetrics endpoint, otherwise returns `OK`. This service check is only available by using the newer OpenMetrics implementation (`use_openmetrics: true`)"
+        "description": "Returns `CRITICAL` if the Agent is unable to connect to the OpenMetrics endpoint, otherwise returns `OK`. This service check is only available with the latest version (`use_openmetrics: true`)."
```

Comment:
```suggestion
        "description": "Returns `CRITICAL` if the Agent is unable to connect to the OpenMetrics endpoint, otherwise returns `OK`. This service check is only available with the latest mode (`use_openmetrics: true`)."
```